package.json

```diff
@@ -326,7 +329,6 @@
     "deep-freeze-strict": "1.1.1",
     "end-of-stream": "^1.4.4",
     "eth-ens-namehash": "^2.0.8",
-    "eth-json-rpc-filters": "^6.0.0",
```

Comment:
i think the pattern has generally been to bump deps (and make any necesarry changes to adopt) in a PR by themselves. Just checking if that applies here